README.asciidoc

```diff
@@ -354,6 +354,8 @@ lsp-inlay-hints-enable global
 
 You can change the hints' face with `set-face global InlayHint <face>`.
 
+Some LSPs support inserting the inlay hints as text into your document. This can be useful when using `basedpyright` to write Python code, since it can make your type hints explicit. To do this, put the primary cursor next to the hint, and run `lsp-inlay-hint-apply-textedit`.
```

Comment:
We're inconsistent about line wrapping,
but I think we should use semantic line breaks (like, one sentence per line),
mostly for easier editing and better diffs
https://nick.groenen.me/notes/one-sentence-per-line/

(also because Kakoune doesn't have great support for softwrapping yet haha)